src/frontend/next/next.config.js

```diff
@@ -20,5 +20,5 @@ if (dotenv.config({ path: envPath }).error) {
 
 module.exports = {
   poweredByHeader: false,
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
I'd leave a comment above this line to indicate that we can't switch over (yet) due to MUI style issues, otherwise someone will do it.